documentation/release_3.1.htm

```diff
@@ -422,7 +422,7 @@ <h3>Other code changes</h3>
 can handle the template.
   All derived classes of <code>CListEvent</code> will need to implement it.</li>
 <li>We removed <code>SeparableMetzArrayFilter::discrete_fourier_transform()</code>. Now <code>stir::fourier</code> is used. No need to comply with the Licence of "Numerical Recipies" anymore.</li>
-</li>
+</li> We added <code>SeparableGaussianArrayFilter</code> and <code>SeparableGaussianImageFilter</code>
```

Comment:
could you move this line up to the "new functionality" section, and add some text like "This filter constructs the kernel by sampling a Gaussian. It is therefore different from the Metz filter with power 0, as that is constructed by sampling a Gaussian in frequency space (and can therefore have negative values in image space)."